common/views/pages/_document.tsx

```diff
@@ -79,6 +80,8 @@ class WecoDoc extends Document<DocumentInitialPropsWithTogglesAndGa> {
     return (
       <Html lang="en">
         <Head>
+          <CivicUK />
```

Comment:
Should be behind the toggle if we're merging, so no merging until a decision is made/the change is applied